### DISCUSSION TITLE
nested fields are not showing up in client

### DISCUSSION THREAD

**Discussion by eriklumbela** [prisma/prisma#6232] [Q&A] [answered]
hi all,

I have an issue trying to delete a double nested field.

this is my data model:
```prisma
model Post {
  id        String     @id @default(uuid())
  title     String
  body      String
  published Boolean @default(false)
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  String?
  comments  Comment[]
}
model User {
  id        String  @id @default(uuid())
  email     String  @unique
  name      String
  age       Int?
  posts     Post[]
  comments  Comment[]
}

model Comment {
  id        String  @id @default(uuid())
  text      String
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  String?
  post      Post?   @relation(fields: [postId], references: [id])
  postId    String?
}
```

when I _update_ a `user`, I also want to _delete_ its `posts` and `comments`
as an extra from the deleted posts i need to delete all `comments`

i was trying to do that with the following query:
```ts
await prisma.user.update({ 
            where: { id },
            data: {
                posts: {
                    updateMany: {
                        data: {
                            ...
                        }
                    },
                }
            }
        })
```

for some reason in the above query a post's comments (and authorId) are not part of the available data?
![Screenshot 2021-03-24 at 16 26 24](https://user-images.githubusercontent.com/28491397/112342055-3a9f6200-8cc2-11eb-85ea-51608efebcca.png)

this makes me think that the schema (or something else) might not be set up correctly
Any suggestions?

**Comment by ryands17** [comment, marked as answer]
@eriklumbela 👋 

Currently `updateMany` doesn't support nested fields: https://github.com/prisma/prisma/issues/3143

As a workaround you can do the following:

```ts
await prisma.post.deleteMany({ where: { authorId: 'id' } })
```

And the comments should automatically be deleted due to a cascade delete. Do follow the issue as well for more updates :)

**Reply by eriklumbela** [reply]
Thanks @ryands17 

In that case I will configure my postgress db to resolve foreign key deletions with action [cascading deletes](https://www.prisma.io/docs/guides/general-guides/database-workflows/cascading-deletes/postgresql) first.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
